src/BeatPlayer.cpp

```diff
@@ -97,7 +98,7 @@ void generateInt16Sine(vector<int16_t>& data, const size_t freq, const double le
 
 
 BeatPlayer::BeatPlayer(const size_t beatRate)
```

Comment:
I checked it https://godbolt.org/z/ei6v6V. It seems the compiler discards the const for value types, but this was also new for me.

Reply:
https://stackoverflow.com/questions/768588/is-the-const-value-parameter-in-definition-but-not-declaration-really-c#768620

Reply:
And it makes also sense for the api side, because the const in your declaration changes nothing for the caller. But at the definition it actualy changes what you can/want to do with the parameter.

Reply:
Yeah, but clang-tidy did removed the `const` in the first place because the parameter is passed by value.  And leaving `const` here is an inconsistency and these make code harder to read.